core/quill.js

```diff
@@ -298,7 +298,6 @@ class Quill {
   }
 }
 Quill.DEFAULTS = {
-  bounds: document.body,
```

Comment:
Can you keep `bounds: null` in the list instead so `DEFAULTS` still has all the possible options listed?